nixosModules/master.nix

```diff
@@ -336,6 +348,18 @@ in
           default = cfg.authBackend == "github";
         };
 
+        userAllowlist = lib.mkOption {
+          type = lib.types.nullOr (lib.types.listOf lib.types.str);
+          default = null;
+          description = "If non-null, specifies users/organizations that are allowed to use buildbot, i.e. buildbot-nix will ignore any repositories not owned by these users/organizations.";
+        };
+
+        repoAllowlist = lib.mkOption {
```

Comment:
Would be cool if you could add those options also to https://github.com/nix-community/buildbot-nix/blob/main/examples/master.nix
To have same example-driven documentation.